README.md

```diff
@@ -6,6 +6,10 @@
 [![License][license-src]][license-href]
 [![Nuxt][nuxt-src]][nuxt-href]
 
+**Built in collaboration with the Durst Organization**
```

Comment:
we could use capital T in the The. cc @hakenprog 

**Built in collaboration with The Durst Organization**